README.md

```diff
@@ -58,6 +59,10 @@ More information can be found here: [Ewon Historical Logging](https://developer.
 
 This connector uses basic authentication (RFC7617) to communicate with OSIsoft and requires basic authentication to be enabled.
 
+### Starting Files
+
+The connector release zip has a starting files directory included in it. This starting files directory contains JVMRUN file and connector configuration file, as well as an OCS specific basic script.
```

Comment:
Why is the connector not in these files in the release?

Reply:
The jar file is available on the releases page as another download. I will add it to the start up directory in the next PR and made a new issue for this.